.github/workflows/e2e-test-weekly.yml

```diff
@@ -376,20 +376,18 @@ jobs:
       max-parallel: 5
       matrix:
         include:
-          - test: "GCP"
-            provider: "gcp"
+          - provider: "gcp"
             regionZone: "europe-west2-a"
-          - test: "AWS"
-            provider: "aws"
+          - provider: "aws"
             regionZone: "us-east-2c"
+          - provider: "azure"
+            regionZone: "northeurope"
```

Comment:
nit: should we rely on the region defaults of the workflow instead of hardcoding them here too?